lib/sdl.cpp

```diff
@@ -831,3 +831,13 @@ void SdlWindow::swapBuffer()
 {
    SDL_GL_SwapWindow(handle->hwnd);
 }
+
+std::string SdlWindow::getHelpString()
+{
+   std::ostringstream oss;
+   auto * fb = std::cout.rdbuf();
+   std::cout.rdbuf(oss.rdbuf());
+   onKeyDown['h'](0);
+   std::cout.rdbuf(fb);
+   return oss.str();
```

Comment:
This works, but I wonder it it will be better to expose the static functions
`SolutionKeyHPressed` in `lib/vssolution.cpp`, `Solution3dKeyHPressed` in `lib/vssolution3d.cpp, and `VectorKeyHPressed` in `lib/vsvector.cpp` and `lib/vsvector3d.cpp`.

@v-dobrev and @tomstitt -- what do you think?

Reply:
Here's another option: https://github.com/GLVis/glvis/pull/142

Reply:
I'd like to hear from @v-dobrev, but I personally like the #142 option more